README.md

```diff
@@ -1,4 +1,4 @@
 ## dsi.cfw.guide
 The complete guide to modding your Nintendo DSi. 
 
-[Link](https://dsi.cfw.guide)
+https://dsi.cfw.guide
```

Comment:
If you want to expand on the README, maybe add something about how to run it locally (ex. [like this section of Universal-DB's](https://github.com/Universal-Team/db#running-the-site-locally)), not really related to this but since you edited the README it made me think of it.

Reply:
What would need to be changed? I've never actually done that.

Reply:
Could maybe do like: (I would make this a suggestion block, but the code blocks are breaking it, here's the markdown: https://mystb.in/EuropeanInterimTechnology.md)

## Running the site locally
To test the website locally, simply run the following commands:
```
gem install bundler jekyll
```
to install jekyll, then
```
bundle install
```
to install the rest of the guide's requirements, then finally
```
bundle exec jekyll serve
```
to build the Markdown files into HTML and start a webserver on http://127.0.0.1:4000.

Reply:
Thanks!